examples/slides/slides.adoc

```diff
@@ -0,0 +1,59 @@
+= AsciiDocJs-PDF Presentation
```

Comment:
nit-pick, the capitalization is "Asciidoctor.js":

* AsciiDoc is the syntax 
* AsciiDoc Python is the deprecated Python implementation
* Asciidoctor is the Ruby implementation (default one now)
* Asciidoctor.js is the JavaScript implementation

Reply:
Note sure what's the proper way to fit PDF in there: choose `Asciidoctor.js PDF` to match https://asciidoctor.org/docs/asciidoctor-pdf/

Reply:
"Asciidoctor.js PDF" is good.
But you could also use a short Rust presentation (if you want) and in this case use a "real" presentation title :smiley:

Reply:
>But you could also use a short Rust presentation (if you want) and in this case use a "real" presentation title smiley

Hehe, the reason why I am researching presentation tools is that I want to give a semester-long Rust course this spring :-)

Right now I am teaching Python, so hopefully I'll make a real presentation by Friday (using slides.com right now).

Reply:
> Hehe, the reason why I am researching presentation tools is that I want to give a semester-long Rust course this spring

I hope Asciidoctor.js PDF will work for you :wink: 

> Right now I am teaching Python, so hopefully I'll make a real presentation by Friday (using slides.com right now :( ).

Feel free to submit improvements to the slide template, I will gladly merge them :+1: